input/fsh/instances/CapabilityStatement-bfdr.fsh

```diff
@@ -10,34 +10,92 @@ Usage: #definition
 * status = #active
 * experimental = false
 * date = "2020-11-01"
-* description = "This section describes the expected capabilities of a BFDR Document producer actor who is responsible for producing clinical documents and a BFDR Document consumer who receives and consumes the clinical documents."
+* description = "This section describes the expected capabilities of a BFDR Document producer actor who is responsible for producing clinical documents. It is expected that the VRO client is able to make the requests supported by the EHR.
+To do this, a VRO (client) will send search requests to a server (EHR) to find particular birth or fetal death reports (Composition). An EHR can also be searched for patients and practitioners to help with searching for birth and fetal death records.
+The Id of a report can then be used to request a document (Bundle) containing the full information of the report (the composition and referenced resources).
+Subsequent transmission of coded and jurisdictional records between jurisdictions and NCHS is left to [FHIR Messaging](https://nightingaleproject.github.io/vital_records_fhir_messaging_ig/v1.0.1/index.html), not described here."
 * kind = #requirements
 * fhirVersion = #4.0.1
 * format[0] = #xml
 * format[+] = #json
-  // These are produced by  the source of the reports and sent to a Jurisdiction VRO or to NCHS
+* rest[0]
+  // To run the operation to get the composition, VRO may want to get particular compositions by subject, author, and date
+  * mode = #server
+  * insert SupportResource(Composition, #SHALL)
+  * insert SupportProfile(http://hl7.org/fhir/us/bfdr/StructureDefinition/Composition-provider-live-birth-report, #SHALL)
+  * insert SupportInteraction(#read, #SHALL)
+  * insert SupportInteraction(#search-type, #SHALL)
+  * insert SupportSearchParam(date, https://hl7.org/fhir/searchparameter-registry.html#clinical-date, #dateTime, #SHALL)
```

Comment:
IG Publisher unhappy with this.... you can sort it out, but probably best to just talk about the story first.

Stack Dump (for debugging):                                                                          (00:00.000 / 01:25.234, 712Mb)
java.lang.Exception: Error parsing /Users/skravitz/git/fhir-bfdr/fsh-generated/resources/CapabilityStatement-CapabilityStatement-bfdr: Unknown SearchParamType code 'dateTime'
	at org.hl7.fhir.igtools.publisher.Publisher.load(Publisher.java:6486)
	at org.hl7.fhir.igtools.publisher.Publisher.loadConformance(Publisher.java:5678)
	at org.hl7.fhir.igtools.publisher.Publisher.createIg(Publisher.java:1134)
	at org.hl7.fhir.igtools.publisher.Publisher.execute(Publisher.java:980)
	at org.hl7.fhir.igtools.publisher.Publisher.main(Publisher.java:12505)
Caused by: java.lang.IllegalArgumentException: Unknown SearchParamType code 'dateTime'
	at org.hl7.fhir.r4.model.Enumerations$SearchParamTypeEnumFactory.fromCode(Enumerations.java:19156)
	at org.hl7.fhir.r4.model.Enumerations$SearchParamTypeEnumFactory.fromCode(Enumerations.java:19133)
	at org.hl7.fhir.r4.formats.JsonParser.parseEnumeration(JsonParser.java:104)
	at org.hl7.fhir.r4.formats.JsonParser.parseCapabilityStatementCapabilityStatementRestResourceSearchParamComponentProperties(JsonParser.java:4371)
	at org.hl7.fhir.r4.formats.JsonParser.parseCapabilityStatementCapabilityStatementRestResourceSearchParamComponent(JsonParser.java:4354)
	at org.hl7.fhir.r4.formats.JsonParser.parseCapabilityStatementCapabilityStatementRestResourceComponentProperties(JsonParser.java:4313)
	at org.hl7.fhir.r4.formats.JsonParser.parseCapabilityStatementCapabilityStatementRestResourceComponent(JsonParser.java:4163)
	at org.hl7.fhir.r4.formats.JsonParser.parseCapabilityStatementCapabilityStatementRestComponentProperties(JsonParser.java:4081)
	at org.hl7.fhir.r4.formats.JsonParser.parseCapabilityStatementCapabilityStatementRestComponent(JsonParser.java:4057)
	at org.hl7.fhir.r4.formats.JsonParser.parseCapabilityStatementProperties(JsonParser.java:3984)
	at org.hl7.fhir.r4.formats.JsonParser.parseCapabilityStatement(JsonParser.java:3786)
	at org.hl7.fhir.r4.formats.JsonParser.parseResource(JsonParser.java:31808)
	at org.hl7.fhir.r4.formats.JsonParserBase.parse(JsonParserBase.java:131)
	at org.hl7.fhir.r4.formats.ParserBase.parse(ParserBase.java:58)
	at org.hl7.fhir.igtools.publisher.Publisher.parseContent(Publisher.java:6977)
	at org.hl7.fhir.igtools.publisher.Publisher.parse(Publisher.java:7064)
	at org.hl7.fhir.igtools.publisher.Publisher.load(Publisher.java:6482)